client/jetpack-connect/plans.jsx

```diff
@@ -188,6 +191,50 @@ class Plans extends Component {
 		);
 	}
 
+	handleInfoButtonClick = info => {
+		this.props.recordTracksEvent( 'calypso_jpc_help_' + info + '_click' );
+	};
+
+	plansExtendedInfo = () => {
+		const { translate } = this.props;
+		const headerText = translate( 'Unsure which plan is right for you?' );
+		const subheaderText = translate(
+			'Read out guide to choosing the right plan or compare plans in more details.'
+		);
+
+		return (
+			<Card className="jetpack-connect__plan-info">
+				<img
+					className="jetpack-connect__plan-info-illustration"
+					src={ '/calypso/images/illustrations/illustration-start-jetpack.svg' }
+					alt=""
+				/>
+				<FormattedHeader
+					className="jetpack-connect__plan-help"
+					headerText={ headerText }
+					subHeaderText={ subheaderText }
+				/>
+				<div className="jetpack-connect__plan-info-buttons">
+					<Button
+						primary
+						href={
+							'https://jetpack.com/2017/02/01/a-simple-guide-to-choosing-the-best-jetpack-plan-your-wordpress-site/'
+						}
+						onClick={ this.handleInfoButtonClick( 'plan_guide' ) }
+					>
+						{ translate( 'PLAN GUIDE' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Plan Guide' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).